mino/minogrpc/tracing/tracers.go

```diff
@@ -0,0 +1,77 @@
+package tracing
+
+import (
+	"io"
+	"sync"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	_ "github.com/uber/jaeger-client-go"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+)
+
+var (
+	ProtocolTag = "protocol"
+)
+
+type key int
+
+const ProtocolKey key = iota
+
+type tracer struct {
```

Comment:
naming: this doesn't represent a `tracer` but a (e.g.) `tracerCatalog` right?

Reply:
Exactly, `catalog` was the word I was looking for!